s3/src/main/resources/reference.conf

```
@@ -74,11 +75,15 @@ alpakka.s3 {
   }
 
   # DEPRECATED (since Alpakka 2.0.0)
-  # AWS S3 is going to retire path-style access, thus prefer the virutal-host-style access
+  # AWS S3 is going to retire path-style access, thus prefer the virtual-host-style access
   # https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/
+  # To switch off the logged warning: use `force`
```

Slightly more verbose docs:

```suggestion
  # Possible values:
  #
  #  - false: use virtual-host style access
  #  - true: use legacy path-style access, warnings will be logged
  #  - force: use legacy path-style access, disable warnings
```